src/components/Userinfo.js

```diff
@@ -1,18 +1,25 @@
 export default class UserInfo {
-  constructor(userName, userJob) {
+  constructor(userName, userJob, userAvatar) {
     this._userName = userName;
     this._userJob = userJob;
+    this._userAvatar = userAvatar;
   }
 
```

Comment:
* `Userinfo.js` shuld be in `camelCase` as well: `UserInfo.js` http://joxi.ru/YmE5JBDtLEn4Pm 

If the file name doesn't change on github then you need to clear `cache` like that

```bash

git rm -r --cached .
git add .
git commit -am 'git cache cleared'
git push

```